src/h5web/visualizations/index.tsx

```diff
@@ -13,21 +12,31 @@ import {
   isDataset,
   isGroup,
 } from '../providers/utils';
-import RawViscontainer from './RawVisContainer';
-import type { VisContainerProps } from './shared/models';
-import ScalarVisContainer from './ScalarVisContainer';
-import MatrixVisContainer from './matrix/MatrixVisContainer';
-import LineVisContainer from './line/LineVisContainer';
-import HeatmapVisContainer from './heatmap/HeatmapVisContainer';
-import NxSpectrumContainer from './nexus/NxSpectrumContainer';
+import RawViscontainer from './containers/RawVisContainer';
+import type { VisContainerProps } from './containers/models';
+import ScalarVisContainer from './containers/ScalarVisContainer';
+import MatrixVisContainer from './containers/MatrixVisContainer';
+import LineVisContainer from './containers/LineVisContainer';
+import HeatmapVisContainer from './containers/HeatmapVisContainer';
+import NxSpectrumContainer from './containers/NxSpectrumContainer';
```

Comment:
We could add an `index.ts` to `containers` to import all `Containers` from `containers`

Reply:
Ah great idea